framework/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <permission
+        android:name="org.eu.droid_ng.wellbeing.framework.permission.BIND"
+        android:protectionLevel="signature|system" />
```

Comment:
make it privilegied too please

Reply:
Yeah, I'll replace `system` to `privileged`, this would require setting the perms in the system xml files tho.

Reply:
that's ok